tests/expecter.go

```diff
@@ -201,9 +201,10 @@ func NewConsoleExpecter(virtCli kubecli.KubevirtClient, vmi *v1.VirtualMachineIn
 	}, timeout, opts...)
 }
```

Comment:
typo in commit message `tests, expecter: improve safe epxecter`: 
the title itself: `epxecter`-->`expecter`
and in the body: `alowes`-->`allows`

Reply:
Done.